content/national-qualifications-framework/ncit/content/thinking-like-a-machine/_index.md

```diff
@@ -43,6 +43,7 @@ prerequisites:
   hard:
   - language-agnostic/problem-solving-technique
   - language-agnostic/intro-to-hackerrank
+  - projects/katas/level-0/
```

Comment:
Please rather use the automarked katas.  They cover the same stuff as katas 0, we just don't need staff to mark them.